src/components/anchor/styles.js

```diff
@@ -0,0 +1,18 @@
+import { colors } from "../../colors/colors";
+
+export const anchor = {
+    fontFamily: "Monospace",
+    fontSize: "1.25rem",
+    fontWeight: "bold",
+    border: "none",
+    borderRadius: "10px",
+    padding: ".4rem 1rem",
+    background: colors.grayTint,
```

Comment:
lets change to `colors.green` and keep the hover background as is, i'll add a new color for this later!